Update todo v1 tutorial

acekyd opened the pull request:
<!--
  For Work In Progress Pull Requests, please use the Draft PR feature,
  see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.
  
  For a timely review/response, please avoid force-pushing additional
  commits if your PR already received reviews or comments.
  
  Before submitting a Pull Request, please ensure you've done the following:
  - 📖 Read the TBD Developer Website Contributing Guide: https://github.com/TBD54566975/developer.tbd.website/blob/main/CONTRIBUTING.md.
  - 📖 Read the TBD Developer Website Code of Conduct: https://github.com/TBD54566975/developer.tbd.website/blob/main/CODE_OF_CONDUCT.md.
  - 👷‍♀️ Create small PRs. In most cases, this will be possible.
  - ✅ Provide tests for your changes.
  - 📝 Use descriptive commit messages.
  - 📗 Update any related documentation and include any relevant screenshots.
-->

## What type of PR is this? (check all applicable)


- [x] 📝 Documentation Update

## Description

<!-- Please do not leave this blank -->

This PR updates the todo app tutorial and resolves #933 

## Related Tickets & Documents
<!-- 
Please use this format link issue numbers: Resolves #123
https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword 
-->
Resolves #

## Mobile & Desktop Screenshots/Recordings

<!-- Visual changes require screenshots -->

## Added code snippets?
- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### No tests? Add a note
<!-- 
If you didn't provide tests with this PR, please explain here why they aren't needed.
-->

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [x] 📓 general documentation
- [ ] 🙅 no documentation needed

### No docs? Add a note
<!-- 
If you didn't provide documentation with this PR, please explain here why it's not needed.
-->

## [optional] Are there any post-deployment tasks we need to perform?



## [optional] What gif best describes this PR or how it makes you feel?



<!-- note: PRs with deletes sections will be marked invalid -->

EbonyLouis reviewed (commented):
**Minor Change Suggestions:** 

**Line 9:** replace "written using `web5.js`". maybe "built using web5,js"
**Line 195:** replace the beginning with "Next, you can add a ToDo to storage by binding an addTodo() method to the UI"
**Line 263:** replace the ending to "..... comment we added earlier"

acekyd commented:
> @acekyd **Minor Change Suggestions:**
> 
> **Line 9:** replace "written using `web5.js`". maybe "built using web5,js" **Line 195:** replace the beginning with "Next, you can add a ToDo to storage by binding an addTodo() method to the UI" **Line 263:** replace the ending to "..... comment we added earlier"

Thanks @EbonyLouis. Updated.
